src/app/keyring/components/AdvKeyGenOptions.js

```diff
@@ -22,7 +23,11 @@ l10n.register([
 
 export default function AdvKeyGenOptions({value: {keyAlgo, keySize, keyExpirationTime}, onChange, disabled}) {
   const context = React.useContext(KeyringOptions);
-  const handleDateChange = moment => onChange({target: {id: 'keyExpirationTime', value: moment}});
+  const tomorrow = addDays(new Date(), 1);
+  const farFuture = addYears(tomorrow, 80);
```

Comment:
Here I am adding 80 years instead of hard-coding the max year to 2080. This kinda extends the default max date to plus 20 years, but this make sense to me, because I guess when designed originally, the idea was to align with a human lifespan of roughly 80 years.